packages/broker-service/src/router/task.ts

```diff
@@ -0,0 +1,22 @@
+import { Router, Request, Response } from "express";
+import { getNextTaskId } from "../domain/get-next-task";
+
+const router = Router();
+
+router.get("/", async (req: Request, res: Response) => {
```

Comment:
Shouldn't mixed RESTful and GraphQL

Reply:
Why not? GrahpQL is on serverless, whereas rest on broker and proof-service, we do not expose them

Reply:
I'm not prefer RESTful due to its lack of documentation.

Reply:
done